apps/judicial-system/web/src/api/index.ts

```diff
@@ -40,18 +40,25 @@ export const getCases: () => Promise<DetentionRequest[]> = async () => {
 export const getCaseById: (
   caseId: string,
 ) => Promise<GetCaseByIdResponse> = async (caseId: string) => {
-  const response = await fetch(`/api/case/${caseId}`)
+  try {
+    const response = await fetch(`/api/case/${caseId}`, {
+      method: 'get',
+      headers: { Authorization: `Bearer ${csrfToken}` },
```

Comment:
Very important - how did we miss this?

Reply:
No idea 🙃  How was this working before this?